swan-lake/resources/featured-scenarios/build-a-data-service-in-ballerina.md

```diff
@@ -426,60 +442,74 @@ import ballerina/http;
 
 service /employees on new http:Listener(8080) {
 
-    isolated resource function post .(@http:Payload Employee emp) returns int|error? {
+    resource function post .(Employee emp) returns int|error? {
         return addEmployee(emp);
     }
     
-    isolated resource function get [int id]() returns Employee|error? {
+    resource function get [int id]() returns Employee|error? {
         return getEmployee(id);
     }
     
-    isolated resource function get .() returns Employee[]|error? {
+    resource function get .() returns Employee[]|error? {
         return getAllEmployees();
     }
     
-    isolated resource function put .(@http:Payload Employee emp) returns int|error? {
+    resource function put .(Employee emp) returns int|error? {
         return updateEmployee(emp);
     }
     
-    isolated resource function delete [int id]() returns int|error? {
+    resource function delete [int id]() returns int|error? {
         return removeEmployee(id);       
     }
 
 }
 ```
 
-## Run the service 
+## Run the service
 
-Execute the command below to run the service.
+Use the [**Run**](/learn/vs-code-extension/run-a-program/) CodeLens of the VS Code extension to build and run the service, as shown below.
 
-```
-$ bal run
-```
+![Run the service](/learn/images/featured-scenarios/build-a-data-service-in-ballerina/run-the-service.gif)
+
+>**Info:** Alternatively, you can run this service by navigating to the project root (i.e., `data_service` directory) and executing the `bal run` command. 
```

Comment:
Package name is different in this guide, but still having `data_service` as directory name here. But ideally, package name should be corrected.   See https://github.com/ballerina-platform/ballerina-dev-website/pull/8674#discussion_r1496511674

Reply:
Changed the package name back to `data-service`.